virttest/qemu_devices/qdevices.py

```diff
@@ -1197,6 +1197,10 @@ def _cmdline_json(self):
             elif val in ('off', 'no', "false") \
                     and key not in expect_string_val:
                 command_dict[key] = False
+            # requested-size from device("driver": "virtio-mem-pci")
+            elif key in ("requested-size"):
```

Comment:
```suggestion
            elif key in ("requested-size", ):
```
for example:
```
>>> key = "aa"
>>> key in ("aaaaa")
True
>>> key in ("aaaaa", )
False
```

Reply:
This really matters, thanks @YongxueHong

Reply:
Done